app/server-utils/stampy.ts

```diff
@@ -199,7 +199,7 @@ const renderText = (text: string | null): string | null => {
   const wrapInDetails = ([chunk, ...rest]: string[]): string => {
     if (!rest || rest.length == 0) return chunk
     return `<div>${chunk}<div>
-           <a href="" class="see-more">See more...</a>
+           <a href="" class="see-more"></a>
```

Comment:
hm, didn't pretter autoformat this into `<a ... />`? I guess no one is going to need to copy the text, so I don't mind doing it in CSS 🤷  .. maybe even better to make the long answer copy-able withtout `See less` 🦩

Reply:
hmm. Turns out that if you copy the text it skips the `See less`, as that's added by CSS. So this works automagically :D